resop.mod

```diff
@@ -0,0 +1,27 @@
+set V; # set of graph nodes
+set S; # set of services
+set E; # graph edges
```

Comment:
Since edges are pairs of vertices, define it as:
```ampl
set E{n1 in N, n2 in N};
```

and lets be tidy. All sets should appear first, then the parameters, later the variables, after that the constraints, and finally the objective function. Kind of like you're already doing.

Reply:
Do the same for service graph edges `set ES`.